SpatialGDK/Source/SpatialGDK/Public/Utils/EntityPool.h

```diff
@@ -35,7 +32,7 @@ class SPATIALGDK_API UEntityPool : public UObject
 	GENERATED_BODY()
 
 public:
-	void Init(USpatialNetDriver* InNetDriver, FTimerManager* TimerManager);
+	void Init(USpatialNetDriver* InNetDriver);
```

Comment:
Potential driveby that are are trying to do in various places - pass this by ref as we don't want this to be nullptr.